teuthology/task/selinux.py

```diff
@@ -116,6 +116,8 @@ def get_denials(self):
         #  http://tracker.ceph.com/issues/14244
         known_denials = [
             'comm="dmidecode"',
+            'comm="setroubleshootd"',
+            'comm="rpm"',
```

Comment:
we already have these regex at line 132 and line 133.

Reply:
thanks @tchaikov, closing this one.